samples/TagHelperSample.Web/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <link href="~/blank.css" rel="stylesheet" />
+    <link href="~/site.min.css" rel="stylesheet"
+          asp-fallback-href="~/site.css"
+          asp-fallback-test-class="hidden"
```

Comment:
Personal opinion: it'd be better to have a different class (or property) to test against so the `asp-fallback-test-class` and `asp-fallback-test-value` don't have the same value. That way we can avoid confusion in the future. I get it now after talking in person but I'd rather have clarity in the sample.